README.md

```diff
@@ -4,7 +4,7 @@
 
   <h3>Stockfish</h3>
 
-  A free and strong UCI chess engine.
+  A free and powerful UCI chess engine.
```

Comment:
This is worded differently because the CLI command `help` outputs text saying `Stockfish is a powerful chess engine for playing and analyzing.`, so it has the word `powerful`, not `strong`. And to be consistent, I have used the same word for the README. Stockfish is not just strong, it is powerful.